src/redux/actions.js

```diff
@@ -164,3 +99,77 @@ export const deleteMovie = movie => ({
 export const deleteList = () => ({
   type: TYPES.DELETE_LIST
 });
+
+//thunk actions
+export const registerUser = (userData, history) => dispatch => {
+  http.users.post.register(userData)
+    .then(() => history.push('/login'))
+    .catch(({ response: { data } }) => {
+      console.log('register err', data);
+      dispatch({
+        type: TYPES.GET_ERRORS,
+        payload: data
+      })
+    })
+};
+
+export const loginUser = (user, history) => dispatch => {
+  http.users.post.login(user) // ! should this be a GET?
```

Comment:
Nope! Post sounds right. You're updating a resouorce (the user) to be logged in, hence a post